docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -473,16 +473,37 @@ relation to port exhaustion.
 - `DISABLE_GRAVATAR`: **false**: Enable this to use local avatars only.
 - `ENABLE_FEDERATED_AVATAR`: **false**: Enable support for federated avatars (see
    [http://www.libravatar.org](http://www.libravatar.org)).
-- `AVATAR_UPLOAD_PATH`: **data/avatars**: Path to store user avatar image files.
-- `REPOSITORY_AVATAR_UPLOAD_PATH`: **data/repo-avatars**: Path to store repository avatar image files.
+
+- `AVATAR_STORE_TYPE`: **local**: Storage type for avatars, `local` for local disk or `minio` for s3 compatible object storage service, default is `local`.
+- `AVATAR_UPLOAD_PATH`: **data/avatars**: Path to store user avatar image files if `AVATAR_STORE_TYPE` is `local`.
+- `AVATAR_SERVE_DIRECT`: **false**: Allows the storage driver to redirect to authenticated URLs to serve files directly. Currently, only Minio/S3 is supported via signed URLs, local does nothing.
+- `AVATAR_MINIO_ENDPOINT`: **localhost:9000**: Minio endpoint to connect only available when `AVATAR_STORE_TYPE` is `minio`
+- `AVATAR_MINIO_ACCESS_KEY_ID`: Minio accessKeyID to connect only available when `AVATAR_STORE_TYPE` is `minio`
+- `AVATAR_MINIO_SECRET_ACCESS_KEY`: Minio secretAccessKey to connect only available when `AVATAR_STORE_TYPE` is `minio`
+- `AVATAR_MINIO_BUCKET`: **gitea**: Minio bucket to store the attachments only available when `AVATAR_STORE_TYPE` is `minio`
+- `AVATAR_MINIO_LOCATION`: **us-east-1**: Minio location to create bucket only available when `AVATAR_STORE_TYPE` is `minio`
+- `AVATAR_MINIO_BASE_PATH`: **avatars/**: Minio base path on the bucket only available when `AVATAR_STORE_TYPE` is `minio`
+- `AVATAR_MINIO_USE_SSL`: **false**: Minio enabled ssl only available when `AVATAR_STORE_TYPE` is `minio`
```

Comment:
We need to avoid this. Imagine what happens we add another storage service.

These settings should just go in to `[storage.avatar]`